ansible/setup.yml

```diff
@@ -84,6 +84,13 @@
       tags:
         - dev-tools
 
+    - name: Increase ulimit
+      pam_limits:
+        domain: epoch
```

Comment:
Shall this rather be `"{{ project_user }}"`?

Reply:
good catch